data/json/npcs/missiondef.json

```
@@ -62,11 +62,11 @@
     "id": "MISSION_GET_ANTIBIOTICS",
     "type": "mission_definition",
     "name": { "str": "Find Antibiotics" },
-    "goal": "MGOAL_FIND_ITEM",
+    "goal": "MGOAL_FIND_ITEM_GROUP",
     "difficulty": 2,
     "value": 150000,
     "urgent": true,
-    "item": "antibiotics",
+    "item": "drugs_antibiotics",
```

```suggestion
    "item_group": "drugs_antibiotics",
```
Ported JSON changes over and tested, it doesn't seem to recognize any of those items as valid. :/

---

we may want to double-test if `MGOAL_FIND_ITEM_GROUP` actually works